lib/unison-prelude/src/Unison/Prelude.hs

```diff
@@ -42,7 +49,15 @@ import GHC.Generics as X (Generic, Generic1)
 import GHC.Stack as X (HasCallStack)
 import Safe as X (atMay, headMay, lastMay, readMay)
 import Text.Read as X (readMaybe)
-import qualified Data.Text as Text
+
+onNothing :: Applicative m => m a -> Maybe a -> m a
```

Comment:
Maybe `whenNothing` for symmetry with `when` and `whenLeft`?

Reply:
`whenNothing` would be `flip onNothing`:

```haskell
whenNothing <thing> <action
```

vs

```haskell
<thing> & onNothing <action>
```